src/components/OSCALMetadata.js

```diff
@@ -33,8 +37,131 @@ const useStyles = makeStyles((theme) => ({
 // Returns a string with a locality-sensitive representation of this date
 const formatDate = (isoDate) => new Date(isoDate).toLocaleString();
 
+function saveModifiedMetadata(modifiedField, modifiableMetadata, newValue) {
+  /* Since the browser url will be of the form
+   * http://localhost:8080/system-security-plans?url=http://localhost:8080/oscal/v1/ssps/{uuid}
+   * and we just want the URL defined in the url parameter of the browser url, we must find the
+   * location of the ? and increment that value by 5 to get the start of the desired url.
+   */
+  const startOfUrl = window.location.href.indexOf("?") + 5;
+  const url = window.location.href.substring(startOfUrl);
+
+  /* We must specify a method of PATCH so the backend service knows we are just
+   * updating a value.
+   *
+   * We also provide a body so we can specify what needs to be modified.
+   */
+  fetch(url, {
+    method: "PATCH",
+    body: JSON.stringify({
+      metadata: {
+        modifiedField: newValue,
+      },
+    }),
+  })
+    .then((res) => res.json())
+    .then(
+      (result) => {
+        /* If we reach this point, then the OSCAL file has successfully been modified
+         * and we want those changes to be visible to the user.
+         */
+        modifiableMetadata["last-modified"][1](
+          formatDate(result["last-modified"])
+        );
+        modifiableMetadata[modifiedField].value[1](newValue);
```

Comment:
I may have missed this in the API spec, but I do want to highlight that this relies on the assumption that the only modified values returned are the (single) field we sent and `last-modified`; and that the field we send isn't a date. That may be fine for now.

What we probably want to do is move the logic for how to display/format/represent as far "down" as we can.

Reply:
The response of the PATCH request is the entire object.  The incoming changes are merged with the existing, the object is persisted, re-fetched from persistence, then returned in the response.

@mikeisen1, the back-end is also responsible for updating the `last-modified` field.

Things will be a bit more complicated here because the top-level component, i.e. `OSCALSsp` should re-render the entire SSP given in the PATCH response, but this metadata component is currently handling the request.